js/symbol/shaping.js

```diff
@@ -123,7 +150,14 @@ function linewrap(shaping, glyphs, lineHeight, maxWidth, horizontalAlign, vertic
                 line++;
             }
 
-            if (breakable[positionedGlyph.codePoint]) {
+            if (positionedGlyph.codePoint > 19968) {
```

Comment:
Magic number alert. We should add a comment about this and/or make it a constant.